2024-01-16 22:30:32,530 (3fa56010) :  INFO (core:349) - Starting framework core - Version: 2.6.3, Build: cc260c4 (Fri Dec 14 20:14:21 UTC 2018)
2024-01-16 22:30:32,534 (3fa56010) :  DEBUG (core:361) - Using the elevated policy
2024-01-16 22:30:32,535 (3fa56010) :  DEBUG (core:450) - Starting runtime component.
2024-01-16 22:30:32,556 (3fa56010) :  DEBUG (core:450) - Starting caching component.
2024-01-16 22:30:32,558 (3fa56010) :  DEBUG (core:450) - Starting data component.
2024-01-16 22:30:32,558 (3fa56010) :  DEBUG (core:450) - Starting networking component.
2024-01-16 22:30:32,561 (3fa56010) :  DEBUG (networking:284) - Loaded HTTP cookies
2024-01-16 22:30:32,562 (3fa56010) :  DEBUG (caching:108) - No info file found, trashing the cache folder
2024-01-16 22:30:32,570 (3fa56010) :  DEBUG (networking:452) - Setting the default network timeout to 20.0
2024-01-16 22:30:32,573 (3fa56010) :  DEBUG (core:450) - Starting localization component.
2024-01-16 22:30:32,574 (3fa56010) :  INFO (localization:409) - Setting the default locale to en-us
2024-01-16 22:30:32,575 (3fa56010) :  DEBUG (core:450) - Starting messaging component.
2024-01-16 22:30:32,579 (3fa56010) :  DEBUG (core:450) - Starting debugging component.
2024-01-16 22:30:32,579 (3fa56010) :  DEBUG (core:450) - Starting services component.
2024-01-16 22:30:32,580 (3fa56010) :  DEBUG (core:450) - Starting myplex component.
2024-01-16 22:30:32,581 (392ff460) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/system/messaging/clear_events/com.plexapp.agents.imdb'
2024-01-16 22:30:32,581 (3fa56010) :  DEBUG (core:450) - Starting notifications component.
2024-01-16 22:30:33,052 (3fa56010) :  DEBUG (accessor:68) - Creating a new model access point for provider com.plexapp.agents.imdb in namespace 'metadata'
2024-01-16 22:30:33,072 (3fa56010) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2024-01-16 22:30:33,089 (3fa56010) :  ERROR (networking:219) - Error opening URL 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2024-01-16 22:30:33,092 (3fa56010) :  CRITICAL (runtime:1299) - Exception getting hosted resource hashes (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 1293, in get_resource_hashes
    json = self._core.networking.http_request("http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes", timeout=10).content
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 265, in content
    return self.__str__()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 243, in __str__
    self.load()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 181, in load
    f = self._opener.open(req, timeout=self._timeout)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 444, in open
    response = meth(req, response)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 557, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 482, in error
    return self._call_chain(*args)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 416, in _call_chain
    result = func(*args)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-cc260c476/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 565, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found

2024-01-16 22:30:33,121 (3fa56010) :  DEBUG (runtime:1117) - Created a thread named 'load_all_services'
2024-01-16 22:30:33,121 (392ff460) :  DEBUG (services:265) - Plug-in is not daemonized - loading services from system
2024-01-16 22:30:33,129 (3fa56010) :  DEBUG (runtime:1117) - Created a thread named 'get_server_info'
2024-01-16 22:30:33,131 (3fa56010) :  DEBUG (core:150) - Finished starting framework core
2024-01-16 22:30:33,132 (3fa56010) :  DEBUG (core:560) - Loading plug-in code
2024-01-16 22:30:33,132 (392ff460) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0J1bmRsZVNlcnZpY2U6QWxsU2VydmljZXM_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMApyMAo_'
2024-01-16 22:30:33,132 (37dff460) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400'
2024-01-16 22:30:34,165 (392ff460) :  DEBUG (services:362) - Loaded services
2024-01-16 22:30:34,210 (37dff460) :  DEBUG (core:538) - Machine identifier is a08675478bee93f590cc67146fe0b3ee6f493f1f
2024-01-16 22:30:34,219 (387ff460) :  DEBUG (services:438) - No shared code to load
2024-01-16 22:30:34,273 (37dff460) :  DEBUG (core:539) - Server version is 1.14.1.5488-cc260c476
2024-01-16 22:30:35,033 (3fa56010) :  DEBUG (core:566) - Finished loading plug-in code
2024-01-16 22:30:35,478 (3fa56010) :  DEBUG (agentkit:1115) - Creating new agent class called PlexMovieAgent
2024-01-16 22:30:35,483 (3fa56010) :  DEBUG (agentkit:933) - Updating agent information: [{'media_types': ['Movie'], 'accepts_from': ['com.plexapp.agents.localmedia'], 'fallback_agent': None, 'contributes_to': ['com.plexapp.agents.themoviedb'], 'languages': ['en', 'sv', 'fr', 'es', 'nl', 'de', 'it', 'da', 'ar', 'ca', 'zh', 'cs', 'et', 'fi', 'el', 'he', 'hi', 'hu', 'id', 'ja', 'ko', 'lv', 'no', 'fa', 'pl', 'pt', 'ro', 'ru', 'sk', 'th', 'tr', 'uk', 'vi'], 'persist_stored_files': True, 'version': 0, 'primary_provider': True, 'prefs': True, 'name': 'Plex Movie'}]
2024-01-16 22:30:35,487 (3fa56010) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0FnZW50U2VydmljZTpVcGRhdGVJbmZv/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQo3CmRpY3QKbGlzdApkaWN0Cmxpc3QKbGlzdApsaXN0Cmxpc3QKMgpzMjMKY29tLnBsZXhhcHAuYWdlbnRzLmltZGJzMTAKaWRlbnRpZmllcnIxCnMxMAphZ2VudF9pbmZvMQpyMgoxMApyMwpzMTEKbWVkaWFfdHlwZXNyNApzMTIKYWNjZXB0c19mcm9tbnMxNApmYWxsYmFja19hZ2VudHI1CnMxNApjb250cmlidXRlc190b3I2CnM5Cmxhbmd1YWdlc2IxczIwCnBlcnNpc3Rfc3RvcmVkX2ZpbGVzaTAKczcKdmVyc2lvbmIxczE2CnByaW1hcnlfcHJvdmlkZXJiMXM1CnByZWZzczEwClBsZXggTW92aWVzNApuYW1lMQpzNQpNb3ZpZTEKczI5CmNvbS5wbGV4YXBwLmFnZW50cy5sb2NhbG1lZGlhMQpzMjkKY29tLnBsZXhhcHAuYWdlbnRzLnRoZW1vdmllZGIzMwpzMgplbnMyCnN2czIKZnJzMgplc3MyCm5sczIKZGVzMgppdHMyCmRhczIKYXJzMgpjYXMyCnpoczIKY3NzMgpldHMyCmZpczIKZWxzMgpoZXMyCmhpczIKaHVzMgppZHMyCmphczIKa29zMgpsdnMyCm5vczIKZmFzMgpwbHMyCnB0czIKcm9zMgpydXMyCnNrczIKdGhzMgp0cnMyCnVrczIKdmlyMAo_'
2024-01-16 22:30:35,509 (3fa56010) :  INFO (core:611) - Started plug-in
2024-01-16 22:30:35,510 (3fa56010) :  DEBUG (socketinterface:160) - Starting socket server
2024-01-16 22:30:35,515 (3fa56010) :  DEBUG (runtime:1117) - Created a thread named 'start'
2024-01-16 22:30:35,516 (3fa56010) :  INFO (socketinterface:184) - Socket server started on port 44821
2024-01-16 22:30:35,517 (3fa56010) :  INFO (pipeinterface:25) - Entering run loop
2024-01-16 22:30:35,518 (3fa56010) :  DEBUG (runtime:717) - Handling request GET /:/prefixes
2024-01-16 22:30:35,522 (3fa56010) :  DEBUG (runtime:814) - Found route matching /:/prefixes
2024-01-16 22:30:35,527 (3fa56010) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 148 bytes
2024-01-16 22:30:35,534 (387ff460) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.imdb/prefs
2024-01-16 22:30:35,542 (392ff460) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.imdb/prefs
2024-01-16 22:30:35,586 (387ff460) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.imdb/prefs
2024-01-16 22:30:35,587 (392ff460) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.imdb/prefs
2024-01-16 22:30:35,589 (387ff460) :  DEBUG (preferences:258) - Loaded preferences from DefaultPrefs.json
2024-01-16 22:30:35,590 (387ff460) :  INFO (preferences:161) - No user preferences file exists
2024-01-16 22:30:35,594 (387ff460) :  DEBUG (preferences:198) - Saved the user preferences
2024-01-16 22:30:35,618 (392ff460) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 2224 bytes
2024-01-16 22:30:35,622 (387ff460) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 2224 bytes
